src/Reflection/ReflectionProperty.php

```diff
@@ -592,9 +671,123 @@ private function computeModifiers(PropertyNode $node): int
         $modifiers  = $node->isReadonly() ? ReflectionPropertyAdapter::IS_READONLY : 0;
         $modifiers += $node->isStatic() ? CoreReflectionProperty::IS_STATIC : 0;
         $modifiers += $node->isPrivate() ? CoreReflectionProperty::IS_PRIVATE : 0;
+        $modifiers += $node->isPrivateSet() ? ReflectionPropertyAdapter::IS_PRIVATE_SET_COMPATIBILITY : 0;
         $modifiers += $node->isProtected() ? CoreReflectionProperty::IS_PROTECTED : 0;
+        $modifiers += $node->isProtectedSet() ? ReflectionPropertyAdapter::IS_PROTECTED_SET_COMPATIBILITY : 0;
         $modifiers += $node->isPublic() ? CoreReflectionProperty::IS_PUBLIC : 0;
+        $modifiers += ($node->flags & ReflectionPropertyAdapter::IS_FINAL_COMPATIBILITY) === ReflectionPropertyAdapter::IS_FINAL_COMPATIBILITY ? ReflectionPropertyAdapter::IS_FINAL_COMPATIBILITY : 0;
+        $modifiers += ($node->flags & Modifiers::ABSTRACT) === Modifiers::ABSTRACT ? ReflectionPropertyAdapter::IS_ABSTRACT_COMPATIBILITY : 0;
 
+        /** @phpstan-ignore return.type */
         return $modifiers;
     }
+
+    private function computeImmediateVirtual(PropertyNode $node): bool
```

Comment:
this method :scream:

Reply:
BTW, I read through it, and this is computing whether the property is "immediate virtual" or not: I'm unsure what that means. Can we give it a better name?

Reply:
We use `immeditate` in other methods to compute info coming from current class. So I've used the same pattern... Naming is hard :D

Reply:
Fair enough :)